doc/properties.org

```diff
@@ -60,7 +60,7 @@
 | Property              | Description                                           | Possible values |
 |-----------------------+-------------------------------------------------------+-----------------|
 | ctia.http.enabled     | enable the http server                                | =true= =false=  |
-| ctia.http.port        | set the listening port                                | number          |
+| ctia.http.port        | set the listening port. 0 for any free port.          | number          |
```

Comment:
Is it desirable in a production environment?

Reply:
Probably doesn't make much sense, I'll make a note.